opentracing_instrumentation/client_hooks/__init__.py

```diff
@@ -28,7 +28,12 @@
 import importlib
 import logging
 
-import six
+from .requests import requests_patcher
```

Comment:
I you sure about this? I remember running into an issue where importing a variable from another module like this creates a local copy in the current module, so if the variable is later re-assigned, the local copy is unaffected. In contrast, referring to it as `requests.requests_patcher` always works correctly (similar to opentracing.tracer).

Reply:
I've removed that import.
Thank you!